### PR TITLE
stephan liveria podcast

### PR DESCRIPTION
**Pull request by derejehinsermu**
I added episode 48 and episode 52

### CODE REVIEW

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+date: 2019-01-20
+episode_link: https://stephanlivera.com/episode/48/
+
+---
+
+media: https://stephanlivera.com/download-episode/955/48.mp3?ref=download
+
+Lacation: Stephan Livera Podcast
+
+Topic: Nicolas Dorier and BTCPayServer – self hosted Bitcoin and Lightning payments
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:15]: This should be the title

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+episode_link: https://stephanlivera.com/episode/48/
+
+---
+
+media: https://stephanlivera.com/download-episode/955/48.mp3?ref=download
+
+Lacation: Stephan Livera Podcast
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:13]: Lacation -> Location

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
+---
+title: btcpayserver 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+episode_link: https://stephanlivera.com/episode/48/
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:7]: Episode_link -> media

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:8]: Rm unneeded space

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 5]
```diff
@@ -0,0 +1,230 @@
+---
+title: btcpayserver 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:5]: Dorie -> Dorier

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+date: 2019-01-20
+episode_link: https://stephanlivera.com/episode/48/
+
+---
+
+media: https://stephanlivera.com/download-episode/955/48.mp3?ref=download
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:11]: Can remove the ?ref=download url parameter

**Review by adamjonas — COMMENTED** [review]
Leaving this on the first file, but most of it applies to the other files as well

**Review by derejehinsermu — COMMENTED** [review]
I updated all

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+media: https://stephanlivera.com/episode/48/
+---
+media: https://stephanlivera.com/download-episode/955/48.mp3
+
+Location: Stephan Livera Podcast
+##
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:12]: Empty header

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
+---
+title: Nicolas Dorier and BTCPayServer – self hosted Bitcoin and Lightning payments 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+media: https://stephanlivera.com/episode/48/
+---
+media: https://stephanlivera.com/download-episode/955/48.mp3
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:9]: This should be the media link in the front matter and not repeated here

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+
+Location: Stephan Livera Podcast
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:11]: This location isn't necessary

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 7 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md]: I added the header.

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 8 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:7]: I changed the download link to media and the episode link to podcast.

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 10]
```diff
@@ -0,0 +1,223 @@
+---
+title: Nicolas Dorier and BTCPayServer – self hosted Bitcoin and Lightning payments 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+media: https://stephanlivera.com/download-episode/955/48.mp3
+---
+podcast: https://stephanlivera.com/episode/48/
+## Introdcution
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:10]: These headers need space around them. This is not valid markdown.

File: stephan-livera-podcast/2019-02-11-jack-dorsey-&-elizabeth-stark.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
+---
+title: Bitcoin- Native Currency of the Internet
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Jack Dorsey and Elizabeth Stark']
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-02-11-jack-dorsey-&-elizabeth-stark.md:5]: these need to be comma separated
```suggestion
speakers: ['Jack Dorsey', 'Elizabeth Stark']
```

File: stephan-livera-podcast/2023-02-23-matt-corallo.md [hunk 12]
```diff
@@ -0,0 +1,197 @@
+---
+title: What Bitcoin Specialises
```
Comment (adamjonas) [inline stephan-livera-podcast/2023-02-23-matt-corallo.md:2]:
```suggestion
title: What Bitcoin Specialises in
```

File: stephan-livera-podcast/2023-02-02-casey-rodarmor.md [hunk 13]
```diff
@@ -0,0 +1,235 @@
+---
+title: Casey Rodarmor What are Ordinals?
```
Comment (adamjonas) [inline stephan-livera-podcast/2023-02-02-casey-rodarmor.md:2]:
```suggestion
title: What are Ordinals?
```

File: stephan-livera-podcast/2023-01-14-james-o’beirne.md [hunk 14]
```diff
@@ -0,0 +1,291 @@
+---
+title: A New Way to HODL?
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['James O’Beirne']
```
Comment (adamjonas) [inline stephan-livera-podcast/2023-01-14-james-o’beirne.md:5]: curly apostrophes are no good
```suggestion
speakers: ['James O'Beirne']
```

Please change the file name and remove the apostrophe all together

File: stephan-livera-podcast/2022-11-13-dhruv-pieter-wuille-and-tim-ruffing.md [hunk 15]
```diff
@@ -0,0 +1,321 @@
+---
+title: v2 P2P Transport Protocol for Bitcoin (BIP324)
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Dhruv, Pieter Wuille and Tim Ruffing']
```
Comment (adamjonas) [inline stephan-livera-podcast/2022-11-13-dhruv-pieter-wuille-and-tim-ruffing.md:5]:
```suggestion
speakers: ['Dhruv', 'Pieter Wuille', Tim Ruffing']
```

File: stephan-livera-podcast/2020-08-09-thomas-v-&-ghost43.md [hunk 16]
```diff
@@ -0,0 +1,689 @@
+---
+title: Electrum Wallet
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Thomas V and Ghost43']
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-08-09-thomas-v-&-ghost43.md:5]:
```suggestion
speakers: ['Thomas V', 'Ghost43']
```

File: stephan-livera-podcast/2020-10-15-nadav kohen-bitcoin-dlcs.md [hunk 17]
```diff
@@ -0,0 +1,373 @@
+---
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-10-15-nadav kohen-bitcoin-dlcs.md:1]: there is a space in the file name

File: stephan-livera-podcast/2021-01-21-wiz-and-simon-mempool.md [hunk 18]
```diff
@@ -0,0 +1,393 @@
+---
+title: Mempool.space – helping Bitcoin migrate to a multi-layer ecosystem
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Wiz and Simon']
```
Comment (adamjonas) [inline stephan-livera-podcast/2021-01-21-wiz-and-simon-mempool.md:5]:
```suggestion
speakers: ['Wiz, 'Simon']
```

File: stephan-livera-podcast/2021-11-16-pieter-wuille-andrew-poelstra-andrew-chow-&-murch.md [hunk 19]
```diff
@@ -0,0 +1,409 @@
+---
+title: SLP321 On-chain scaling with Bitcoin Core Developers
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Pieter Wuille,Andrew Poelstra,Andrew Chow & Murch']
```
Comment (adamjonas) [inline stephan-livera-podcast/2021-11-16-pieter-wuille-andrew-poelstra-andrew-chow-&-murch.md:5]:
```suggestion
speakers: ['Pieter Wuille', 'Andrew Poelstra', 'Andrew Chow', 'Mark Erhardt']
```

File: stephan-livera-podcast/2021-11-27-lisa-neigut-alex-bosworth-graham-krizek-and-matt-corallo.md [hunk 20]
```diff
@@ -0,0 +1,409 @@
+---
+title: SLP324 Scaling Bitcoin off-chain with Matt Corallo, Alex Bosworth, Lisa Neigut, and Graham Krizek
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Lisa Neigut, Alex Bosworth, Graham Krizek and Matt Corallo']
```
Comment (adamjonas) [inline stephan-livera-podcast/2021-11-27-lisa-neigut-alex-bosworth-graham-krizek-and-matt-corallo.md:5]:
```suggestion
speakers: ['Lisa Neigut', 'Alex Bosworth', 'Graham Krizek', 'Matt Corallo']
```

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md [hunk 21]
```diff
@@ -0,0 +1,191 @@
+---
+title: Half Signature Aggregation-What is it and how does it help scale Bitcoin?
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Jonas Nick & Tim Ruffing']
```
Comment (adamjonas) [inline stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md:5]:
```suggestion
speakers: ['Jonas Nick', 'Tim Ruffing']
```

File: stephan-livera-podcast/2023-01-23-antoine-poinsot-and-salvatore-ingala.md [hunk 22]
```diff
@@ -0,0 +1,319 @@
+---
+title: SLP452 Bitcoin Miniscript and what it enables – with Antoine Poinsot & Salvatore Ingala
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Antoine Poinsot and Salvatore Ingala']
```
Comment (adamjonas) [inline stephan-livera-podcast/2023-01-23-antoine-poinsot-and-salvatore-ingala.md:5]:
```suggestion
speakers: ['Antoine Poinsot', 'Salvatore Ingala']
```

File: stephan-livera-podcast/2020-08-28-stepan-snigirev-and-ben-kaufman.md [hunk 23]
```diff
@@ -0,0 +1,393 @@
+---
+title: Specter Desktop Bitcoin Multi Sig
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Stepan Snigirev and Ben Kaufman']
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-08-28-stepan-snigirev-and-ben-kaufman.md:5]:
```suggestion
speakers: ['Stepan Snigirev', 'Ben Kaufman']
```

File: stephan-livera-podcast/2021-09-29-jestopher-and-tony.md [hunk 24]
```diff
@@ -0,0 +1,325 @@
+---
+title: Becoming A Lightning Routing Node Operator
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Jestopher and Tony']
```
Comment (adamjonas) [inline stephan-livera-podcast/2021-09-29-jestopher-and-tony.md:5]:
```suggestion
speakers: ['Thomas Jestopher', 'Anthony Potdevin']
```

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 10 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:11]: Thanks. I updated according to your suggestion

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 5 above)
Comment (adamjonas) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:5]: this still needs correction

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md [hunk 25]
```diff
@@ -0,0 +1,225 @@
...
+title: Nicolas Dorier and BTCPayServer – self hosted Bitcoin and Lightning payments 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+media: https://stephanlivera.com/download-episode/955/48.mp3
+---
+podcast: https://stephanlivera.com/episode/48/
+
+## Introdcution
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:11]:
```suggestion
## Introduction
```

File: stephan-livera-podcast/2019-02-11-jack-dorsey-&-elizabeth-stark.md [hunk 26]
```diff
@@ -0,0 +1,209 @@
+---
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-02-11-jack-dorsey-&-elizabeth-stark.md:1]: remove `&` from file name

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 10 above)
Comment (adamjonas) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md:11]: why do we need these headers at all? They don't appear to add anything to the transcript

File: stephan-livera-podcast/2019-11-16-electrum-wallet.md [hunk 27]
```diff
@@ -0,0 +1,255 @@
+---
+title: Electrum Wallet-Bitcoin seeds, PSBT, Multi sig, Lightning
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: [' Thomas Voegtlin']
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-11-16-electrum-wallet.md:5]:
```suggestion
speakers: ['Thomas Voegtlin']
```

File: stephan-livera-podcast/2020-08-09-thomas-v-&-ghost43.md [hunk 28]
```diff
@@ -0,0 +1,689 @@
+---
+title: Electrum Wallet
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Thomas V', 'Ghost43']
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-08-09-thomas-v-&-ghost43.md:5]:
```suggestion
speakers: ['Thomas Voegtlin', 'Ghost43']
```

File: stephan-livera-podcast/2020-08-09-thomas-v-&-ghost43.md [hunk 29]
```diff
@@ -0,0 +1,689 @@
+---
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-08-09-thomas-v-&-ghost43.md:1]: remove `&` from file name

File: stephan-livera-podcast/2020-01-06-andreas-m-antonopoulos.md [hunk 30]
```diff
@@ -0,0 +1,183 @@
+---
+title: Mastering Lightning & Using BTCPayServer
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Andreas M. Antonopoulos']
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-01-06-andreas-m-antonopoulos.md:5]:
```suggestion
speakers: ['Andreas Antonopoulos']
```

File: stephan-livera-podcast/2020-04-07-gleb-naumenko-erlay.md [hunk 31]
```diff
@@ -0,0 +1,344 @@
...
+speakers: ['Gleb Naumenko']
+date: 2020-04-07
+media: https://stephanlivera.com/download-episode/1909/164.mp3
+---
+podcast: https://stephanlivera.com/episode/164/
+
+## Introduction
+
+Gleb Naumenko of Chaincode Labs joins me in this episode to talk about erlay.
+Stephan Livera:
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-04-07-gleb-naumenko-erlay.md:14]:
```suggestion
Stephan Livera:

Gleb Naumenko of Chaincode Labs joins me in this episode to talk about erlay.
```

File: stephan-livera-podcast/2020-04-21-chris-belcher.md [hunk 32]
```diff
@@ -0,0 +1,496 @@
+---
+title: What’s The Problem With Bitcoin Surveillance?
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Chris Belcher']
+date: 2020-04-21
+media: https://stephanlivera.com/download-episode/1955/167.mp3?ref=download
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-04-21-chris-belcher.md:7]:
```suggestion
media: https://stephanlivera.com/download-episode/1955/167.mp3
```

File: stephan-livera-podcast/2020-05-27-bitcoin-coin-selection.md [hunk 33]
```diff
@@ -0,0 +1,371 @@
+---
+title: Bitcoin Coin Selection - Managing Large Wallets
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Murch joins']
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-05-27-bitcoin-coin-selection.md:5]:
```suggestion
speakers: ['Mark Erhardt']
```

File: stephan-livera-podcast/2020-05-27-bitcoin-coin-selection.md [hunk 34]
```diff
@@ -0,0 +1,371 @@
...
+date: 2020-05-27
+media: https://www.youtube.com/watch?v=B3wn6JC0aWc
+---
+podcast: https://stephanlivera.com/episode/177/
+
+## Transcript:
```
Comment (adamjonas) [inline stephan-livera-podcast/2020-05-27-bitcoin-coin-selection.md:11]: why transcript vs introduction?

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md [hunk 35]
```diff
@@ -0,0 +1,191 @@
+---
```
Comment (adamjonas) [inline stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md:1]: rm `&` from filename

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md [hunk 36]
```diff
@@ -0,0 +1,191 @@
...
+title: Half Signature Aggregation-What is it and how does it help scale Bitcoin?
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Jonas Nick', 'Tim Ruffing']
+date: 2022-08-02
+media: https://www.youtube.com/watch?v=sUpITup2igE
+---
+podcast: https://stephanlivera.com/episode/400/
+## Transcript
```
Comment (adamjonas) [inline stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md:10]: space

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md [hunk 37]
```diff
@@ -0,0 +1,191 @@
...
+
+Tim Ruffing [54:11]: Yeah this is an interesting thing maybe to point out: this is only true for full aggregation, but that full aggregation would incentivize you to do CoinJoins, because now you can save transaction fees. Because, while running the CoinJoin, you anyway need some interaction and you would then also do the full signature aggregation, and then the resulting transaction would be more compact. And in particular, for large CoinJoins, the effect is then larger. So there would be an incentive to do CoinJoins, which is a pretty nice thing.
+
+Stephan Livera: Yep. And so it would just depend on which model is used, what wallet, and how it all works, and dealing with the interactivity part of that. And so maybe for some people they would say, Oh that’s too much effort for me! But other people might say, No actually it’s worth it for me — I’ll deal with the interactivity aspects of it because I’m getting a saving, and maybe there’s a bit of privacy involved there too.
+
+Tim Ruffing [55:18]: It also gives you a good argument why you [should] do CoinJoins: because it saves fees — simple as that!
```
Comment (adamjonas) [inline stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md:169]: let's make sure this formatting and timestamps are consistent with the other transcripts

File: stephan-livera-podcast/2023-01-14-james-a-new-way-to-hodl.md [hunk 38]
```diff
@@ -0,0 +1,291 @@
+---
```
Comment (adamjonas) [inline stephan-livera-podcast/2023-01-14-james-a-new-way-to-hodl.md:1]: file name should be stephan-livera-podcast/2023-01-14-james-obeirne-a-new-way-to-hodl.md

**Review by adamjonas — COMMENTED** [review]
I didn't review every file but from what I did see, there needs to be further corrections before this is ready for merging.

- Make sure filename just have alphanumeric characters
- I think you can remove the header that's been added to the top of each transcript. It isn't adding any context.
- The speakers need to be reviewed for accuracy

File: stephan-livera-podcast/2023-01-14-james-a-new-way-to-hodl.md (hunk 38 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2023-01-14-james-a-new-way-to-hodl.md]: the filename is renamed.

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md (hunk 37 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md]: I just copy and past the transcript.
Some files have timestamps with transcript. Should I remove the time stamps?
let me know.

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md (hunk 36 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md]: I removed all the header.

File: stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md (hunk 35 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2022-08-02-jonas-nick-&-tim-ruffing.md]: I remove the '&' and renamed the file file name

File: stephan-livera-podcast/2020-05-27-bitcoin-coin-selection.md (hunk 34 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2020-05-27-bitcoin-coin-selection.md]: I remove all the headers

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 25 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md]: I removed all the headers

File: stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md (hunk 5 above)
Comment (derejehinsermu) [reply stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md]: Oh, sorry 
I made correction. Thanks

**Comment by derejehinsermu** [comment]
> I didn't review every file but from what I did see, there needs to be further corrections before this is ready for merging.
> 
>     * Make sure filename just have alphanumeric characters
> 
>     * I think you can remove the header that's been added to the top of each transcript. It isn't adding any context.
> 
>     * The speakers need to be reviewed for accuracy

I checked all files two times.
I removed all the headers.

File: stephan-livera-podcast/2019-12-27-aj-towns–schnorr-taproot-tapscript-bips.md [hunk 39]
```diff
@@ -0,0 +1,303 @@
+---
+title: Schnorr Taproot Tapscript BIPs 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['AJ Towns']
```
Comment (adamjonas) [inline stephan-livera-podcast/2019-12-27-aj-towns–schnorr-taproot-tapscript-bips.md:5]:
```suggestion
speakers: ['Anthony Towns']
```

File: stephan-livera-podcast/2021-08-24-seedsigner.md [hunk 40]
```diff
@@ -0,0 +1,337 @@
+---
+title: Bitcoin multi sig security under $50
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['SeedSigner']
```
Comment (adamjonas) [inline stephan-livera-podcast/2021-08-24-seedsigner.md:5]:
```suggestion
```

File: stephan-livera-podcast/2021-11-07-t-bast.md [hunk 41]
```diff
@@ -0,0 +1,337 @@
+---
+title: Lightning Protocol Privacy Exploration
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['T-Bast']
```
Comment (adamjonas) [inline stephan-livera-podcast/2021-11-07-t-bast.md:5]:
```suggestion
speakers: ['Bastien Teinturier']
```

File: stephan-livera-podcast/2023-01-23-antoine-poinsot-and-salvatore-ingala.md [hunk 42]
```diff
@@ -0,0 +1,319 @@
+---
+title: SLP452 Bitcoin Miniscript and what it enables – with Antoine Poinsot & Salvatore Ingala
```
Comment (adamjonas) [inline stephan-livera-podcast/2023-01-23-antoine-poinsot-and-salvatore-ingala.md:2]:
```suggestion
title: Bitcoin Miniscript and what it enables
```

**Pull request merged**